.golangci.yml

```diff
@@ -8,7 +8,6 @@ linters:
     - depguard
     - dogsled
     - dupl
-    - exportloopref
```

Comment:
Removed this linter, it's disabled in 1.24:
```
WARN The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar.
ERRO [linters_context] exportloopref: This linter is fully inactivated: it will not produce any reports.
```